src/me/mrCookieSlime/Slimefun/api/TickerTask.java

```diff
@@ -136,15 +134,31 @@ public void run() {
 														stream.println("  Build: " + Bukkit.getVersion());
 														stream.println("  Minecraft: " + Bukkit.getBukkitVersion());
 														stream.println();
-														stream.println("Installed Plugins (" + Bukkit.getPluginManager().getPlugins().length + ")");
+														stream.println("Slimefun Environment:");
+														stream.println("  CS-CoreLib v" + CSCoreLib.getLib().getDescription().getVersion());
+														stream.println("  Slimefun v" + SlimefunStartup.instance.getDescription().getVersion());
+														stream.println();
+
+														List<String> plugins = new ArrayList<>();
+														List<String> addons = new ArrayList<>();
 														for (Plugin p: Bukkit.getPluginManager().getPlugins()) {
 															if (Bukkit.getPluginManager().isPluginEnabled(p)) {
-																stream.println("  + " + p.getName() + " " + p.getDescription().getVersion());
+																plugins.add("  + " + p.getName() + " " + p.getDescription().getVersion());
+																if (p.getDescription().getDepend().contains("Slimefun") || p.getDescription().getSoftDepend().contains("Slimefun"))
+																	addons.add("  + " + p.getName() + " " + p.getDescription().getVersion());
 															}
 															else {
-																stream.println("  - " + p.getName() + " " + p.getDescription().getVersion());
+																plugins.add("  - " + p.getName() + " " + p.getDescription().getVersion());
+																if (p.getDescription().getDepend().contains("Slimefun") || p.getDescription().getSoftDepend().contains("Slimefun"))
+																	addons.add("  - " + p.getName() + " " + p.getDescription().getVersion());
 															}
 														}
+
+														stream.println(" Installed Addons (" + addons.size() + ")");
+														for (String addon : addons) stream.println(addon);
+														stream.println();
+														stream.println("Installed Plugins (" + plugins.size() + "):");
+														for (String plugin : plugins) stream.println(plugin);
```

Comment:
I would rather always use brackets for loops, makes it more readable in my opinion.